### PR TITLE
Enable SSL in Presto

### PR DESCRIPTION
**Pull request by jeff303**
Expose SSL port from Presto and add useful information on how to fetc…

### CODE REVIEW

**Comment by jeff303** [comment]
Do not merge until change to Docker image is merged: metabase/presto-mb-ci#2

File: run-presto.sh
Comment (dpsutton) [inline run-presto.sh:8]: do you still need 8080 in here?

Comment (dpsutton) [reply run-presto.sh:8]: nevermind. i see

**Review by dpsutton — APPROVED** [review]

**Pull request merged**
